### ISSUE TITLE
tiny topic open for report several bugs on every versions been released

### ISSUE THREAD

**Issue by boykensoidong** [RoderickQiu/wnr#97]
Nap setting (Notify you to take a nap for several seconds every several minutes when working.)
[turn On] Nap setting = error at counting/timming sceen (mess-up)

happen only in wnr-1.30.0-patch version 
https://youtu.be/EHobi80raas

**Comment by RoderickQiu** [comment]
Indeed! This is a crazy bug QwQ and just fixed up. Just because I am not really using that feature currently...
